### ISSUE TITLE
MLP Plot graph

### ISSUE THREAD

**Issue by tiimoS** [literally-a-placeholder/pattern_recognition#13]
Plot a graph showing the error on the training set and the validation set, respectively, with respect to the training epochs. Plot showing the accuracy and loss on the training and the validation set with respect to the training epochs.
